Libplanet/Store/StoreExtension.cs

```diff
@@ -1,6 +1,7 @@
 using System;
-using System.Collections.Generic;
+#pragma warning disable S1128 // Remove this unnecessary 'using'
 using System.Collections.Immutable;
+#pragma warning restore S1128 // Remove this unnecessary 'using'
```

Comment:
Why not just remove this 'using' but ignore?

Reply:
This is to suppress a false error(https://github.com/SonarSource/sonar-dotnet/issues/2694). It is resolved(https://github.com/SonarSource/sonar-dotnet/pull/2798) but not released yet. We can remove this warning suppression after the release.